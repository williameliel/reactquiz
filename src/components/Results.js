import React from 'react';

function Results({ score, length, verdict }) {

    const result = `You scored ${score} out of ${length}. ${verdict}`;

    return (
        <h2>
          { result }     
        </h2>
    );
};

export default Results;
