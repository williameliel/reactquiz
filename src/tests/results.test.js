import React from 'react';
import Results from '../components/Results';
import renderer from 'react-test-renderer';

it('Results renders correctly', () => {
  const tree = renderer.create(
    <Results />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});