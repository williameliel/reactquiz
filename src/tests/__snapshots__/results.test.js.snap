exports[`test Results renders correctly 1`] = `
<h2>
  You scored undefined out of undefined. undefined
</h2>
`;
